config/routes.rb

```diff
@@ -7,4 +7,6 @@
   post '/login', to: 'sessions#create'
   
   root to: "items#index"
+
+  delete '/logout', to: 'sessions#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.